portfolio_aboutme.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <meta charset='UTF-8'>
+	<meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    <title>Jake Rooms - About me</title>
+	<link rel='stylesheet' href='normalize.css'/>
+	<link rel='stylesheet' href='portfolio.css'/>
+</head>
+<body>
+	<!--Html for the top navigation bar-->
+	<nav class='link_bar'>
+		<a class='top_link' href='portfolio_home.html'>Home</a>
+	</nav>
+	<!--Html for the title-->
+	<div class='center_bar'>
+		<h1>About me</h1>
+	</div>
+	<!--Html for section 1 with text on the left-->
+	<section class='paragraph'>
+		<h3>Intro</h3>
+		<!--Flexbox for the text and image, changes to column-reverse after a media queery-->
+		<div class='text_box left'>
+			<p>Hi I'm Jake! I'm a mathematics graduate from the university of sheffield and a software developer in training. Gaming and reading are some of my hobbies. I'm hoping to get a career that will be a fun challenge and let me grow as a person. I belive that I have a powerful combination of proffesional skills as coding and mathematics work well together, and in the curent enviroment pure mathematics such as Topology is making its way into software development.</p>
+			<img class='img' src='jake_face.jpg' alt='my face'/>
+		</div>
+	</section>
+	<!--Html for section 2 with text on the right-->
+	<section class='paragraph'>
+		<h3>Event 2</h3>
+		<!--Flexbox for the text and image, changes to column after a media queery-->
+		<div class='text_box right'>
+			<img class='img' src='jake_face.jpg' alt='my face'/>
+			<p>"On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble that are bound to ensue; and equal blame belongs to those who fail in their duty through weakness of will, which is the same as saying through shrinking from toil and pain. These cases are perfectly simple and easy to distinguish. In a free hour, when our power of choice is untrammelled and when nothing prevents our being able to do what we like best, every pleasure is to be welcomed and every pain avoided. But in certain circumstances and owing to the claims of duty or the obligations of business it will frequently occur that pleasures have to be repudiated and annoyances accepted. The wise man therefore always holds in these matters to this principle of selection: he rejects pleasures to secure other greater pleasures, or else he endures pains to avoid worse pains."</p>
+		</div>
+	</section>
+	<!--html for a center break section and final section-->
+	<div class='center_bar'>
+		<h2>Event 3</h2>
```

Comment:
Still not sure why your third event has H2 tag when your intro/event 2 have H3 tags?

Reply:
formatting and the centre bar will be used as a break